cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -315,6 +322,7 @@ protected void getPartitions(@Nullable PartitionFilter filter, PartitionConsumer
         }
         byte[] pathBytes = row.get(RELATIVE_PATH);
         if (pathBytes != null) {
+          byte[] creationTime = row.get(CREATION_TIME);
```

Comment:
right, the time is now being read in `metadataFromRow` method. Removed this unnecessary